doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,19 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		func _ready():
+		  var spin_box = SpinBox.new()
+		  get_tree().get_root().add_child(spin_box)
+		  var line_edit = $SpinBox.get_line_edit()
```

Comment:
This is incorrect, the `SpinBox` node you instanced was not given the "SpinBox" name, and you put it under root, so you can't access it via `$SpinBox` which would fetch a node named "SpinBox" in the current scene.

Since you already have a reference to the `SpinBox` instance in `spin_box`, you can simply use that instead, so `spin_box.get_line_edit()`.